JASP-Engine/JASP/R/multinomialtestbayesian.R

```diff
@@ -219,24 +219,24 @@ MultinomialTestBayesian <- function(jaspResults, dataset, options, ...) {
   nhyps <- multinomialResults[["mainTable"]][["nhyps"]]
   nms   <- multinomialResults[["specs"]][["hypNames"]]
 
-  descriptivesTable$addColumnInfo(name = "fact",   title = factorVariable, type = "string", combine = TRUE)
-  descriptivesTable$addColumnInfo(name="observed", title = "Observed",     type = numberType)
+  descriptivesTable$addColumnInfo(name = "fact",     title = factorVariable,      type = "string", combine = TRUE)
+  descriptivesTable$addColumnInfo(name = "observed", title = gettext("Observed"), type = numberType)
 
   # If no variable is selected, adjust column title
   if(is.null(nhyps)){
-    descriptivesTable$addColumnInfo(name = "expected", title = "Expected", type = numberType)
+    descriptivesTable$addColumnInfo(name = "expected", title = gettext("Expected"), type = numberType)
   } else if(nhyps == 1) {
-      descriptivesTable$addColumnInfo(name = nms, title = paste0("Expected: ", nms), type = numberType)
+      descriptivesTable$addColumnInfo(name = nms, title = paste0(gettext("Expected: "), nms), type = numberType)
   } else if(nhyps > 1) {
     for(h in 1:nhyps){
-      descriptivesTable$addColumnInfo(name = nms[h], title = nms[h], type = numberType, overtitle = "Expected")
+      descriptivesTable$addColumnInfo(name = nms[h], title = nms[h], type = numberType, overtitle = gettext("Expected"))
     }
   }
 
   if (ciRequested) {
-      descriptivesTable$addColumnInfo(name = "lowerCI", title = "Lower", type = "number",
+      descriptivesTable$addColumnInfo(name = gettext("lowerCI"), title = "Lower", type = "number",
                                       overtitle = paste0(100 * ciInterval, "% ", ciColumnName))
-      descriptivesTable$addColumnInfo(name = "upperCI", title = "Upper", type = "number",
+      descriptivesTable$addColumnInfo(name = gettext("upperCI"), title = "Upper", type = "number",
```

Comment:
gettext() should be over title instead of name

Reply:
![](https://i.giphy.com/3otPotnGlA5FSF2fKw.gif)